continuous-integration.md

```diff
@@ -31,7 +32,7 @@ Every minute you reduce your building time is a minute saved when you will need
 
 ## Platforms
 
-* Use TeamCity to run continuous integration tasks
+* Use TeamCity or GitHub Actions (with [`guardian/actions-assume-aws-role`]) to run continuous integration tasks
 * Where possible, have CI execute a single, centralised script in the repository named `script/ci` 
     - This adheres to GitHub's [Scripts To Rule Them All pattern]
 * You can use [GitHub Actions] for most above tasks, however there is not currently a departmental best practice for uploading artifacts to RiffRaff through it.
```

Comment:
This is no longer true.

Reply:
I clearly didn't read the entire file! Updated now.